--- a/icu/source/config/mh-mingw64	2013-04-22 21:01:19.792718771 +0200
+++ b/icu/source/config/mh-mingw64	2013-04-22 21:01:43.990527291 +0200
@@ -72,7 +72,7 @@
 ## An import library is needed for z/OS and MSVC
 IMPORT_LIB_EXT = .lib
 
-LIBPREFIX=
+LIBPREFIX= lib
 
 # Change the stubnames so that poorly working FAT disks and installation programs can work.
 # This is also for backwards compatibility.
