--- source/tools/toolutil/toolutil.cpp.orig	2014-01-22 20:05:23.000000000 +0100
+++ source/tools/toolutil/toolutil.cpp	2014-04-18 14:48:19.255758414 +0200
@@ -51,9 +51,11 @@
 #   include <sys/types.h>
 #endif
 
-/* In MinGW environment, io.h needs to be included for _mkdir() */
+/* In MinGW environment, io.h needs to be included for _mkdir() for
+ * the mingw.org toolchain and direct.h for the mingw-w64 toolchain */
 #if U_PLATFORM == U_PF_MINGW
 #include <io.h>
+#include <direct.h>
 #endif
 
 #include <errno.h>
