--- source/configure.ac.orig	2013-12-09 18:32:25.000000000 +0100
+++ source/configure.ac	2014-04-18 14:41:18.357144597 +0200
@@ -649,6 +649,12 @@
 ia64-*-hpux*)
     GENCCODE_ASSEMBLY="-a aCC-ia64"
     ;;
+i686-*-mingw32)
+    GENCCODE_ASSEMBLY="-a gcc-mingw-x86"
+    ;;
+x86_64-*-mingw32)
+    GENCCODE_ASSEMBLY="-a gcc-mingw-x64"
+    ;;
 esac
 AC_SUBST(GENCCODE_ASSEMBLY)
 
--- source/config/mh-mingw.orig	2014-04-18 14:35:49.866027927 +0200
+++ source/config/mh-mingw	2014-04-18 14:41:18.357144597 +0200
@@ -135,9 +135,6 @@
 # Environment variable to set a runtime search path
 LDLIBRARYPATH_ENVVAR = PATH
 
-# The type of assembly to write for generating an object file
-GENCCODE_ASSEMBLY=-a gcc-cygwin
-
 # These are needed to allow the pkgdata GNU make files to work
 PKGDATA_DEFS = -DU_MAKE=\"$(MAKE)\"
 
--- source/config/mh-mingw64.orig	2014-04-18 14:46:31.745371250 +0200
+++ source/config/mh-mingw64	2014-04-18 14:46:39.393327725 +0200
@@ -136,9 +136,6 @@
 # Environment variable to set a runtime search path
 LDLIBRARYPATH_ENVVAR = PATH
 
-# The type of assembly to write for generating an object file
-GENCCODE_ASSEMBLY=-a gcc-mingw64
-
 # These are needed to allow the pkgdata GNU make files to work
 PKGDATA_DEFS = -DU_MAKE=\"$(MAKE)\"
 
