--- a/zeromq-3.2.2/configure	2013-05-14 13:39:06.014730402 -0700
+++ b/zeromq-3.2.2/configure	2013-07-01 16:48:36.086687142 -0700
@@ -17650,9 +17650,9 @@
         libzmq_on_mingw32="yes"
         libzmq_dso_visibility="no"
 
-        if test "x$enable_static" = "xyes"; then
-            as_fn_error $? "Building static libraries is not supported under MinGW32" "$LINENO" 5
-        fi
+#        if test "x$enable_static" = "xyes"; then
+#            as_fn_error $? "Building static libraries is not supported under MinGW32" "$LINENO" 5
+#        fi
         ;;
     *cygwin*)
         # Define on Cygwin to enable all library features
@@ -17660,9 +17660,9 @@
 
 $as_echo "#define ZMQ_HAVE_CYGWIN 1" >>confdefs.h
 
-        if test "x$enable_static" = "xyes"; then
-            as_fn_error $? "Building static libraries is not supported under Cygwin" "$LINENO" 5
-        fi
+#        if test "x$enable_static" = "xyes"; then
+#            as_fn_error $? "Building static libraries is not supported under Cygwin" "$LINENO" 5
+#        fi
         ;;
     *)
         as_fn_error $? "unsupported system: ${host_os}." "$LINENO" 5
