--- a/zeromq-3.2.2/include/zmq.h	2012-11-13 17:09:14.000000000 -0800
+++ b/zeromq-3.2.2/include/zmq.h	2013-05-02 14:25:40.514494594 -0700
@@ -37,6 +37,8 @@
 #endif
 
 /*  Handle DSO symbol visibility                                             */
+#define ZMQ_EXPORT
+/* bda zzzzzzzzzzzzzzzzzzzzzz
 #if defined _WIN32
 #   if defined DLL_EXPORT
 #       define ZMQ_EXPORT __declspec(dllexport)
@@ -52,6 +54,7 @@
 #       define ZMQ_EXPORT
 #   endif
 #endif
+*/
 
 /******************************************************************************/
 /*  0MQ versioning support.                                                   */
